environment.yml

```diff
@@ -29,5 +29,5 @@ dependencies:
     - pytest-xdist
     - pexpect
 variables:
-  CHANNELS: -c defaults -c numba -c intel -c numba/label/dev -c dppy/label/dev --override-channels
-  CHANNELS_DEV: -c dppy/label/dev -c defaults -c numba -c intel -c numba/label/dev --override-channels
+  CHANNELS: -c defaults -c numba -c intel -c numba/label/dev --override-channels
```

Comment:
I think `dppy/label/dev` has no need to remove.
I think dpnp does not control something else i.e. dpcpp-cpp-rt.
Could you print please what packages from this channels installed if not removing it?